partner_industry_secondary/models/res_config_settings.py

```diff
@@ -12,3 +13,9 @@ class ResConfigSettings(models.TransientModel):
         help="Set if you want to be able to use industries for individuals also.",
         implied_group="partner_industry_secondary.group_use_partner_industry_for_person",
     )
+    display_last_child_first = fields.Boolean(
+        string="Display Child Industries first",
+        help="Set if you want to show the last child industries first "
+        "when searching for industries.\nChild (Dad < Grampa < Ancestor)",
```

Comment:
issue: this affects display, not search AFAICS. Besides, let's be a bit more formal here 😁 

```suggestion
        "when displaying industries.\nChild (Parent < Grandparent)",
```